app/assets/javascripts/mirador_serializer.es6

```diff
@@ -66,7 +66,7 @@ export default class MiradorSerializer {
       case 1:
         return '1x1';
       case 2:
-        return this.twoUpLayout();
+        return this.constructor.twoUpLayout();
```

Comment:
I'm surprised we would need this here and not in the other branches of the case statement.

Reply:
Made this a non-instance method to make eslint happy: https://github.com/sul-dlss/vatican_exhibits/pull/259/files#diff-ff7d05cf051991986c2af5a43fb07dd0R83

Reply:
![no_idea_what_i_m_doing](https://user-images.githubusercontent.com/96776/42718449-cf641be6-86bd-11e8-9762-4e331c1c17a0.jpg)

Reply:
![](https://i.imgflip.com/2dz8re.jpg)